app/src/screens/Onboarding/SignInScreen.tsx

```diff
@@ -14,7 +35,41 @@ type SignInScreenProps = StackScreenProps<OnboardingStackParams, 'SignInScreen'>
 
 export const SignInScreen = ({ navigation }: SignInScreenProps) => {
   return (
-    <FullScreenWhite>
+    <SignInMainContainer>
+      <SignInHeaderContainer>
+        <LogoContainer />
+        <SignInHeaderTextContainer>
+          <SignInHeaderTitleText>Universal Store</SignInHeaderTitleText>
```

Comment:
We have the same text style for the header and the "Sign In" at the top of the form. That's why I combined the two. Same for sub-header text